frontend/src/components/NutritionFacts.jsx

```diff
@@ -0,0 +1,141 @@
+        for (let i = 0; i < dataArray.length; i++) {
+            calories += Math.round(dataArray[i][0]["ENERC_KCAL"]["quantity"]);
+
+            sumNutrition["Total Fat"]["quantity"] += Math.round(dataArray[i][0]["FAT"]["quantity"]);
+            sumNutrition["Total Fat"]["dailyValue"] += Math.round(dataArray[i][1]["FAT"]["quantity"]);
+
+            sumNutrition["Cholesterol"]["quantity"] += Math.round(dataArray[i][0]["CHOLE"]["quantity"]);
+            sumNutrition["Cholesterol"]["dailyValue"] += Math.round(dataArray[i][1]["CHOLE"]["quantity"]);
+
+            sumNutrition["Sodium"]["quantity"] += Math.round(dataArray[i][0]["NA"]["quantity"]);
+            sumNutrition["Sodium"]["dailyValue"] += Math.round(dataArray[i][1]["NA"]["quantity"]);
+
+            sumNutrition["Carbohydrates"]["quantity"] += Math.round(dataArray[i][0]["CHOCDF"]["quantity"]);
+            sumNutrition["Carbohydrates"]["dailyValue"] += Math.round(dataArray[i][1]["CHOCDF"]["quantity"]);
+
+            sumNutrition["Protein"]["quantity"] += Math.round(dataArray[i][0]["PROCNT"]["quantity"]);
+            sumNutrition["Protein"]["dailyValue"] += Math.round(dataArray[i][1]["PROCNT"]["quantity"]);
+
+            sumNutrition["Vitamin-D"]["quantity"] += Math.round(dataArray[i][0]["VITD"]["quantity"]);
+            sumNutrition["Vitamin-D"]["dailyValue"] += Math.round(dataArray[i][1]["VITD"]["quantity"]);
+
+            sumNutrition["Calcium"]["quantity"] += Math.round(dataArray[i][0]["CA"]["quantity"]);
+            sumNutrition["Calcium"]["dailyValue"] += Math.round(dataArray[i][1]["CA"]["quantity"]);
+
+            sumNutrition["Iron"]["quantity"] += Math.round(dataArray[i][0]["FE"]["quantity"]);
+            sumNutrition["Iron"]["dailyValue"] += Math.round(dataArray[i][1]["FE"]["quantity"]);
+
+            sumNutrition["Potassium"]["quantity"] += Math.round(dataArray[i][0]["K"]["quantity"]);
+            sumNutrition["Potassium"]["dailyValue"] += Math.round(dataArray[i][1]["K"]["quantity"]);
+
+            sumNutrition["Vitamin-C"]["quantity"] += Math.round(dataArray[i][0]["VITC"]["quantity"]);
+            sumNutrition["Vitamin-C"]["dailyValue"] += Math.round(dataArray[i][1]["VITC"]["quantity"]);
+        }
+    }
+
+    function handleNutritionData(recipe){
+        sumNutritionData("Green Machine Smoothie")
+    }
+
+    const keys = Object.keys(sumNutrition)
+    const sumValues = Object.values(sumNutrition)
+    return (
+
+        <div className="nutritionCard">
+            <h1>Nutrition Facts</h1>
+            <h3>Serving Size <p>1 shake</p></h3>
+            <div className="caloriesDiv"><h2>Calories</h2><h2>{calories}</h2></div>
+            <div className="dailyValue"><div></div><h4 >%Daily Value*</h4></div>
+             <ul>
+                {keys.map((key, index) => <li key={index}><div className="listDiv"><h4>{key}</h4>{sumValues[index]["quantity"]}{sumValues[index]["unit"]}</div><p>{sumValues[index]["dailyValue"]}%</p></li>)}
```

Comment:
Same deal here, can you put the elements on separate lines so it's easier to see what's going on, especially with the `sumValues` stuff.
```suggestion
                {keys.map((key, index) => (
                    <li key={index}>
                        <div className="listDiv">
                            <h4>{key}</h4>
                            {sumValues[index]["quantity"]}{sumValues[index]["unit"]}
                        </div>
                        <p>{sumValues[index]["dailyValue"]}%</p>
                    </li>
                ))}
```